src/views/sent_sms_list/SentSmsList.module.css

```diff
@@ -7,7 +7,3 @@
     align-items: flex-end;
     margin-bottom: var(--spacers-dp16);
 }
-
-.headerRight {
-    margin-left: auto;
-}
```

Comment:
It seems that the delete button is now left aligned because of this change. That introduces a bit of a discrepancy between the layout of both routes, as in the other route it is right aligned. I was wondering why this was changed.

Reply:
I was trying to make the CSS of both the sent and receive lists more similar and I forgot to add `flex-grow: 1` to the single select field on the sent SMS list.